frontend/controllers/scanctrl.py

```diff
@@ -420,20 +419,15 @@ def set_result(scanid, file_hash, probe, result):
             pr = _fetch_probe_result(fw, probe)
             _update_ref_results(fw, fw.file, pr)
             fw.file.update(session=session)
-            # init empty NoSql record to get
-            # all mandatory fields initialized
-            prr = ProbeRealResult()
-            # and fill with probe raw results
-            prr.update(sanitized_res)
-            # link it to Sql record
-            pr.nosql_id = prr.id
+            # fill ProbeResult with probe raw results
+            pr.doc = sanitized_res
             pr.status = sanitized_res.get('status', None)
             s_type = sanitized_res.get('type', None)
             pr.type = IrmaProbeType.normalize(s_type)
             pr.update(session=session)
             probedone = []
             for pr in fw.probe_results:
-                if pr.nosql_id is not None:
+                if pr.doc is not None:
```

Comment:
my comment is for from line [435](https://github.com/quarkslab/irma-frontend/pull/126/files#diff-0a24ed5d3f543bec8dc62ab60600680cL435) to [437](https://github.com/quarkslab/irma-frontend/pull/126/files#diff-0a24ed5d3f543bec8dc62ab60600680cL437) included